packages/react-components/src/organisms/UserProfileRecentWorks.tsx

```diff
@@ -129,26 +136,58 @@ const UserProfileRecentWork: React.FC<UserProfileRecentWorkProps> = ({
 
 const UserProfileRecentWorks: React.FC<UserProfileRecentWorksProps> = ({
   orcidWorks = [],
-}) => (
-  <Card>
-    <div css={headerStyles}>
-      <Headline2 styleAsHeading={3}>
-        Recent Publications ({orcidWorks.length})
-      </Headline2>
-      <Paragraph accent="lead">Via ORCID</Paragraph>
-      <span css={{ display: 'grid', svg: { fill: lead.rgb } }}>
-        {orcidIcon}
-      </span>
-    </div>
-    <ul css={listStyles}>
-      {orcidWorks
-        .flatMap((work, idx) => [
-          <Divider key={`sep-${idx}`} />,
-          <UserProfileRecentWork key={`wrk-${idx}`} {...work} />,
-        ])
-        .slice(1)}
-    </ul>
-  </Card>
-);
+  orcid,
+}) => {
+  const { isOwnProfile } = useContext(UserProfileContext);
+
+  return orcidWorks.length || isOwnProfile ? (
+    <Card>
+      <div css={headerStyles}>
+        <Headline2 styleAsHeading={3}>
+          Recent Publications ({orcidWorks.length})
+        </Headline2>
+        <Paragraph accent="lead">Via ORCID</Paragraph>
+        <span css={{ display: 'grid', svg: { fill: lead.rgb } }}>
+          {orcidIcon}
+        </span>
+      </div>
+      {orcidWorks.length === 0 && isOwnProfile ? (
+        <Paragraph accent="lead">
+          <span css={titleStyle}>No works available on your ORCID.</span>
+          <br />
+          To complete this section, please add works to your ORCID.{' '}
+          <span css={{ svg: { stroke: fern.rgb } }}>
+            <Link href="https://support.orcid.org/hc/en-us/articles/360006973133-Add-works-to-your-ORCID-record">
+              Learn how to add works.
+              <span css={{ verticalAlign: 'middle' }}>{externalLinkIcon}</span>
+            </Link>
+          </span>
+          <br />
+          <br />
+          <span css={titleStyle}>Your ORCID is: </span>
+          <Link href={new URL(`https://orcid.org/${orcid}`).toString()}>
+            {orcid}
+          </Link>
+          <br /> If this is incorrect please{' '}
+          <Link href={mailToSupport({ subject: 'Orcid change request' })}>
```

Comment:
```suggestion
          <Link href={mailToSupport({ subject: 'ORCID change request' })}>
```
Unless this was specified by requirements

Reply:
It was copy pasted from the trello